jenkins/jobs/lxd-test-storage-disks-vm.yaml

```diff
@@ -0,0 +1,36 @@
+- job:
+    name: "lxd-test-storage-disks-vm"
+    concurrent: false
+    description: Run VM storage disk tests on physical hardware.
+    node: master
+    project-type: freestyle
+
+    builders:
+    - shell: |-
+        cd /lxc-ci
+        exec sudo /lxc-ci/bin/maas-run tags=physical focal hwe-20.04 iommu bin/test-lxd-storage-disks-vm
```

Comment:
is `hwe-20.04` sufficient to run the test on the HWE kernel?

Reply:
should be, yeah